src/content/docs/en/core-concepts/astro-components.mdx

```diff
@@ -291,7 +291,7 @@ Named slots can also be passed to [UI framework components](/en/core-concepts/fr
 
 
 :::note
-An astro slot name can not be dynamically generated, such as within a map function. If this feature is needed within UI framework components, it might be best to generate these dynamic slots within the framework itself.
+To dynamically generate an astro slot name, such as within a map function, you need use Astro versions later than 4.2.0. If this feature is needed within UI framework components on older Astro versions, it might be best to generate these dynamic slots within the framework itself.
```

Comment:
```suggestion
Dynamically generating an astro slot name, such as within a map function, was added in Astro v4.2.0. For earlier versions of Astro, it might be best to generate these dynamic slots within the framework itself.
```

Reply:
Typically, we write docs for the current version, so ideally I wouldn't want this as a note at all. If this is possible now, I think an example of usage would be fabulous, and the better way to go in addition to a note like the above.

BUT, for a super quick fix, I would just amend the note this way, and am happy to approve this.

Reply:
Sounds good, I'll add an example usage of the feature on top of this note.

Reply:
Yeah it should I'll edit that @VoxelMC, thanks!